Tests/TuistGeneratorTests/Generator/WorkspaceGeneratorTests.swift

```diff
@@ -125,12 +128,68 @@ final class WorkspaceGeneratorTests: XCTestCase {
         XCTAssertEqual(cocoapodsInteractor.installArgs.count, 1)
     }
 
+    func test_generate_addsPackageDependencyManager() throws {
+        // Given
+        let target = anyTarget(dependencies: [
+            .package(.remote(url: "http://some.remote/repo.git", productName: "Example", versionRequirement: .exact("branch"))),
+        ])
+        let project = Project.test(path: fileHandler.currentPath,
+                                   name: "Test",
+                                   settings: .default,
+                                   targets: [target])
+        let graph = Graph.create(project: project,
+                                 dependencies: [(target, [])])
+
+        let workspace = Workspace.test(name: project.name,
+                                       projects: [project.path])
+        let workspacePath = path.appending(component: workspace.name + ".xcworkspace")
+        system.succeedCommand(["xcodebuild", "-resolvePackageDependencies", "-workspace", workspacePath.pathString, "-list"])
+        try fileHandler.createFiles(["\(workspace.name).xcworkspace/xcshareddata/swiftpm/Package.resolved"])
+
+        // When
+        try subject.generate(workspace: workspace,
+                             path: fileHandler.currentPath,
+                             graph: graph,
+                             tuistConfig: .test())
+
+        // Then
+        XCTAssertTrue(fileHandler.exists(path.appending(component: "Package.resolved")))
+
+        XCTAssertNoThrow(try subject.generate(workspace: workspace,
+                                              path: fileHandler.currentPath,
+                                              graph: graph,
+                                              tuistConfig: .test()))
+    }
+
+    func test_generate_doesNotAddPackageDependencyManager() throws {
+        // Given
+        let target = anyTarget()
+        let project = Project.test(path: path,
+                                   name: "Test",
+                                   settings: .default,
+                                   targets: [target])
+        let graph = Graph.create(project: project,
+                                 dependencies: [(target, [])])
+
+        let workspace = Workspace.test(projects: [project.path])
+
+        // When
+        try subject.generate(workspace: workspace,
+                             path: path,
+                             graph: graph,
+                             tuistConfig: .test())
+
+        // Then
+        XCTAssertFalse(fileHandler.exists(path.appending(component: ".package.resolved")))
```

Comment:
Shouldn't stick to the same naming and call this file `Package.resolved`?

Reply:
I wanted to hide it from the user and for example if you use danger, you need `Package.resolved` in the root which would conflict with this file.

Reply:
Got you! Then let's keep it.

Reply:
Let's update the version in WorkspaceGenerator.swift as I changed that to `Packaged.resolved` - let's go back to `.Package.resolved`

Reply:
Updated 👍